alf/drivers/off_policy_driver.py

```diff
@@ -195,32 +201,26 @@ def train(self,
         experience = self._algorithm.preprocess_experience(experience)
 
         length = experience.step_type.shape[1]
-        if mini_batch_length is None:
-            mini_batch_length = length
-        else:
-            mini_batch_length = min(mini_batch_length, length)
+        mini_batch_length = (mini_batch_length or length)
+        assert length % mini_batch_length == 0, (
+            "length=%s mini_batch_length=%s" % (length, mini_batch_length))
```

Comment:
be a bit more informative in the error message?  e.g. length=%s not a multiple of mini_batch_length=%s.

Reply:
updated